src/components/general/DatePicker.js

```diff
@@ -143,68 +94,59 @@ class DatePicker extends Component {
 					/>
 				</View>
 			);
-		else
+		}
+		else {
 			return (
-				<View>
-					<View style = { datePickerStyle }>
-						<View style = { fieldContainer }>
-							<PickerInput
-								data = { monthArr }
-								style = { style }
-								title = { "Enter a Month" }
-								inputBoxStyle = { inputBoxStyle }
-								dropDownArrowStyle = { dropDownArrowStyle }
-								iconSize = { iconSize }
-								iconColor = 'black'
-								value = { month }
-								onSelect = { (text) => this.clickActionMonth(text) }
-								placeholder = { "MM" }
-							/>
-						</View>
-						<View style = { fieldContainer }>
-							<PickerInput
-								data = { dayArr }
-								style = { style }
-								title = { "Enter a Day" }
-								inputBoxStyle = { inputBoxStyle }
-								dropDownArrowStyle = { dropDownArrowStyle }
-								iconSize = { iconSize }
-								iconColor = 'black'
-								value = { day }
-								onSelect = { (text) => this.clickActionDay(text) }
-								placeholder = { "DD" }
-							/>
-						</View>
-						<View style = { fieldContainer }>
-							<PickerInput
-								data = { yearArr }
-								style = { style }
-								title = { "Enter a Year" }
-								inputBoxStyle = { inputBoxStyle }
-								iconSize = { iconSize }
-								iconColor = 'black'
-								dropDownArrowStyle = { dropDownArrowStyle }
-								value = { year }
-								onSelect = { (text) => this.clickActionYear(text) }
-								placeholder = { "YYYY" }
-							/>
-						</View>
+				<View style = { datePickerStyle }>
+					<View style = { fieldContainer }>
+						<PickerInput
+							data = { monthArray }
+							style = { style }
+							title = { "Enter a Month" }
+							inputBoxStyle = { inputBoxStyle }
```

Comment:
Props could be saved in an object,
```suggestion
 defaultProps = {
   	inputBoxStyle:  inputBoxStyle ,
   	dropDownArrowStyle: dropDownArrowStyle,
        iconSize: iconSize,
	iconColor : 'black' 
 }
 
 <PickerInput  { ...defaultProps } and everything else/>
```

Reply:
Title can also be removed

Reply:
And write Title without curly braces